src/single-spa-layout.js

```diff
@@ -1,4 +1,3 @@
-export { constructRoutes } from "./constructRoutes.js";
-export { matchRoute } from "./matchRoute.js";
```

Comment:
Removed this intentionally - it reduces the bundle size and isn't really needed in the browser.

Reply:
#188 @joeldenning